modules/core/include/opencv2/core/hal/intrin_avx512.hpp

```diff
@@ -1509,6 +1509,21 @@ inline v_int8x64 v_absdiffs(const v_int8x64& a, const v_int8x64& b)
 inline v_int16x32 v_absdiffs(const v_int16x32& a, const v_int16x32& b)
 { return v_max(a, b) - v_min(a, b); }
 
+static inline v_uint8x64 v_absdiffs(const v_uint8x64& a, const v_uint8x64& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
+
+static inline v_uint16x32 v_absdiffs(const v_uint16x32& a, const v_uint16x32& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
+
+static inline v_float32x16 v_absdiffs(const v_float32x16& a, const v_float32x16& b)
+{
+    return v_max(a, b) - v_min(a, b);
+}
```

Comment:
template  ?

Reply:
I can't apply templates because there are v_absdiffs() function with different implementation. For example:

```
inline v_int8x16 v_absdiffs(const v_int8x16& a, const v_int8x16& b)
{
    v_int8x16 d = a - b;
    v_int8x16 m = a < b;
    return (d ^ m) - m;
}
```